internal/service/alarms/internal/db/migrations/000003_create_alarm_event_record_table.up.sql

```diff
@@ -39,21 +38,23 @@ CREATE TABLE IF NOT EXISTS alarm_event_record (
 -- Set ownership of the alarm_sequence_seq sequence to alarm_event_record.alarm_sequence_number
 ALTER SEQUENCE alarm_sequence_seq OWNED BY alarm_event_record.alarm_sequence_number;
 
--- Function to update the alarm_sequence_number on specific status or time changes
-CREATE OR REPLACE FUNCTION update_alarm_event_sequence()
+CREATE OR REPLACE FUNCTION update_alarm_event_record()
     RETURNS TRIGGER AS $$
 BEGIN
-    -- Update sequence if status changes to 'resolved' or if alarm_changed_time is updated
-    IF (NEW.alarm_status = 'resolved' AND OLD.alarm_status IS DISTINCT FROM 'resolved')
-       OR (NEW.alarm_changed_time IS DISTINCT FROM OLD.alarm_changed_time) THEN
+    -- Update sequence and fields for resolved status
```

Comment:
Please do this in code. We had some discussions with Allain about this and the conclusion was that triggers are a pain to maintain. Let's avoid them and do stuff in code

Reply:
removed and using code